docs/img-src/action-sequence.mmd

```diff
@@ -0,0 +1,24 @@
+sequenceDiagram
+  participant mocha
+  participant Detox Server
+  participant Detox Client
+  participant Earl Grey
+
+  %% Note over mocha,Detox Server: Runs on your computer
+  %% Note over Detox Client,Earl Grey: Runs on the device
+
+  mocha->>Detox Server: element(by.label('Click Me')).tap();
```

Comment:
detox is unopinionated re test runner, change mocha to "test runner"
Detox server is actually something else. `detox-server` is a different module that runs a web socket server, letting both tester (detox logic inside a test runner) and testee (detox code inside a device) communicate with each other. Both tester and testee are websocket clients